qiita_db/support_files/populate_test_db.sql

```diff
@@ -440,7 +440,8 @@ INSERT INTO qiita.collection_job (collection_id, job_id) VALUES (1, 1);
 INSERT INTO qiita.collection_users (email, collection_id) VALUES ('shared@foo.bar', 1);
 
 --add default analysis for users
-INSERT INTO qiita.analysis (email, name, description, dflt, analysis_status_id) VALUES ('test@foo.bar', 'test@foo.bar-dflt', 'dflt', true, 1), ('admin@foo.bar', 'admin@foo.bar-dflt', 'dflt', true, 1), ('shared@foo.bar', 'shared@foo.bar-dflt', 'dflt', true, 1), ('demo@microbio.me', 'demo@microbio.me-dflt', 'dflt', true, 1), ('test@foo.bar', 'test@foo.bar-dflt', 'dflt', true, 2), ('admin@foo.bar', 'admin@foo.bar-dflt', 'dflt', true, 2), ('shared@foo.bar', 'shared@foo.bar-dflt', 'dflt', true, 2), ('demo@microbio.me', 'demo@microbio.me-dflt', 'dflt', true, 2);
+INSERT INTO qiita.analysis (email, name, description, dflt, analysis_status_id) VALUES ('test@foo.bar', 'test@foo.bar-dflt-1', 'dflt', true, 1), ('admin@foo.bar', 'admin@foo.bar-dflt-1', 'dflt', true, 1), ('shared@foo.bar', 'shared@foo.bar-dflt-1', 'dflt', true, 1), ('demo@microbio.me', 'demo@microbio.me-dflt-1', 'dflt', true, 1), ('test@foo.bar', 'test@foo.bar-dflt-2', 'dflt', true, 1), ('admin@foo.bar', 'admin@foo.bar-dflt-2', 'dflt', true, 2), ('shared@foo.bar', 'shared@foo.bar-dflt-2', 'dflt', true, 2), ('demo@microbio.me', 'demo@microbio.me-dflt-2', 'dflt', true, 2);
+INSERT INTO qiita.analysis_portal (analysis_id, portal_type_id) VALUES (3, 1), (4, 1), (5, 1), (6, 1), (7, 2), (8, 2), (9, 2), (10, 2);
```

Comment:
If I understand correctly, what you're doing here is making the default analysis only visible on their portal, right? If so, I think this is correct.